src/dsp_tools/models/value.py

```diff
@@ -26,11 +26,18 @@ def __str__(self) -> str:
     def get_all_iris(self) -> Optional[list[str]]:
         return self.__iriregexp.findall(self.__xmlstr)
 
+    def find_ids_referenced_in_salsah_links(self) -> set[str]:
+        return set(regex.findall(pattern='href="IRI:(.*?):IRI"', string=self.__xmlstr))
+
     def replace(self, fromStr: str, toStr: str) -> None:
         self.__xmlstr = self.__xmlstr.replace(fromStr, toStr)
 
-    def regex_replace(self, pattern: str, repl: str) -> None:
-        self.__xmlstr = regex.sub(pattern=repr(pattern)[1:-1], repl=repl, string=self.__xmlstr)
+    def replace_one_id_with_iri_in_salsah_link(self, internal_id: str, iri: str) -> None:
```

Comment:
Sorry to complain again, but it should be salsah_links (plural). 
(my fault that I see it only now)